src/nordictrackelliptical.cpp

```diff
@@ -397,47 +397,39 @@ void nordictrackelliptical::forceResistance(resistance_t requestResistance) {
             writeCharacteristic((uint8_t *)res22, sizeof(res22), QStringLiteral("resistance22"), false, true);
             break;
         }
-    } else if (!proform_hybrid_trainer_xt) {
+    } else if (proform_hybrid_trainer_xt) {
```

Comment:
why did you modifiy this case? You should change the nordictrack_elliptical_c7_5 case only above

Reply:
I think it is better to have an if structure like this:
if (nordictrack_elliptical_c7_5) {
} else if (proform_hybrid_trainer_xt) {
] else if (new_futur_nordictrack_proform_elliptical) {
} else {
}

Reply:
yes I agree with you but you modififed the content of the proform_hybrid_trainer_xt instead of the content of nordictrack_elliptical_c7_5 case. Am I wrong?

Reply:
I didn't touch the content of the proform_hybrid_trainer_xt, I have swapped the content of proform_hybrid_trainer_xt with the content of the else with a simple copy/paste to change !proform_hybrid_trainer_xt into proform_hybrid_trainer_xt. GitHub makes this simple swap so hard to read...
The only thing I have added in proform_hybrid_trainer_xt was case 15 and case 16 in switch (requestResistance) because they are missing from your original file.

Reply:
hah sorry, yes github makes me think that you swapped them :)